czech/admin/config_post_icons.lang.php

```diff
@@ -16,20 +16,20 @@
 $l['manage_post_icons'] = "Spravovat ikony příspěvků";
 $l['manage_post_icons_desc'] = "V této sekci můžete editovat, mazat a spravovat ikony příspěvků.";
 
-$l['name_desc'] = "Tohle je název ikony příspěvku.";
+$l['name_desc'] = "Toto je název ikony příspěvku.";
```

Comment:
Jestli se nepletu a zároveň se nic nezměnilo, tak toto je hovorový výraz
Na základě: https://github.com/Eldenroot/MyBB_CZE-translation-1.8.x/commit/c25e6b53021b02e853fe0c7e4a55018870dd3fa5

Reply:
Podle Ústavu pro jazyk český je hovorové "tohle". 
https://prirucka.ujc.cas.cz/?slovo=toto 
https://prirucka.ujc.cas.cz/?slovo=tohle

Reply:
Aha, tak to se moc omlouvám. Zrovna tohle je konkrétní případ jak jsou commit message důležité. :facepalm: